.gitignore

```diff
@@ -3,4 +3,5 @@
 .vscode
 
 node_modules/
-.eslintrc.*
\ No newline at end of file
+.eslintrc.*
```

Comment:
Why don't we want .eslintrc? I'd have thought it should be here for CI and so we're all following the same standards

Reply:
I swear this was added in automatically LMAO. I'll remove this.